src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,32 @@
+.section-title {
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: -1.5px;
+  margin: 0;
+
+  @include onTable {
+    font-size: 40px;
+  }
+
+  @include onDesktop {
+    font-size: 48px;
+  }
+}
+
+.section-title__position {
+  text-align: center;
+  font-size: inherit;
```

Comment:
The 'font-size: inherit;' seems unnecessary as it will automatically inherit the font-size from its parent. If you want to be explicit about the inheritance you can leave it, but normally it is not needed.